services/src/handlers/plots.rs

```diff
@@ -56,39 +61,60 @@ pub(crate) struct GetPlot {
 ///   "operator": {
 ///     "type": "Statistics",
 ///     "params": {},
-///     "rasterSources": [
-///       {
-///         "type": "MockRasterSource",
-///         "params": {
-///           "data": [
-///             {
-///               "time": {
-///                 "start": -8334632851200000,
-///                 "end": 8210298412799999
-///               },
-///               "tilePosition": [0, 0],
-///               "globalGeoTransform": {
-///                 "originCoordinate": { "x": 0.0, "y": 0.0 },
-///                 "xPixelSize": 1.0,
-///                 "yPixelSize": -1.0
-///               },
-///               "gridArray": {
-///                 "shape": {
-///                   "shapeArray": [3, 2]
+///     "sources": {
+///       "source": [
+///         {
+///           "type": "MockRasterSourcei32",
+///           "params": {
+///             "data": [
+///               {
+///                 "time": {
+///                   "start": -8334632851200000,
+///                   "end": 8210298412799999
+///                 },
+///                 "tilePosition": [0,0],
+///                 "globalGeoTransform": {
+///                   "originCoordinate": {"x": 0.0,"y": 0.0},
+///                   "xPixelSize": 1.0,
+///                   "yPixelSize": -1.0
+///                 },
+///                 "gridArray": {
+///                   "type": "grid",
+///                   "innerGrid": {
+///                     "shape": {
+///                       "shapeArray": [3, 2]
+///                     },
+///                     "data": [1, 2, 3, 4, 5, 6]
+///                   },
+///                   "validityMask": {
```

Comment:
Multiple examples are not supported by utoipa at the moment. As you can see [in the docs](https://docs.rs/utoipa/latest/utoipa/openapi/schema/struct.Object.html#structfield.example), the example field can only contain one value.

I filed [an issue on GitHub](https://github.com/juhaku/utoipa/issues/447).

Reply:
What do you think. Merge this and wait for the new version to complete the task? Or wait here?

Reply:
I think we should leave this task open and put it on the list "Extern geblockt". If we merge this now we could forget that we wanted to do something else here too.

Reply:
I saw that in the meantime the ability to set multiple examples got released: [utoipa-gen-3.0.0 ](https://github.com/juhaku/utoipa/releases/tag/utoipa-gen-3.0.0).
I tested what happens if I update utoipa from 2.4 to 3.0 and i get many compilation errors. Because of that I think we should update utoipa and add the examples in another PR to keep things cleaner.